test/tasks/example/sep/013-gnosis-safe-approve-hash/config.toml

```diff
@@ -0,0 +1,18 @@
+l2chains = [{ name = "Base Sepolia", chainId = 84532 }]
+
+safeAddressString = "BaseNested"
+safeTxHash = "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+templateName = "GnosisSafeApproveHash"
+
+[addresses]
+  # BaseNested is a 2/2 between BaseSafe and BaseSC. When using this config file
+  # as a template, uncomment the correct section for the chain you're using.
+  # --- Sepolia ---
+  BaseNested = "0x646132A1667ca7aD00d36616AFBA1A28116C770A"
+  BaseSC = "0x5dfEB066334B67355A15dc9b67317fD2a2e1f77f"
+  BaseSafe = "0x6AF0674791925f767060Dd52f7fB20984E8639d8"
```

Comment:
I haven't yet found where I need to change code to support signing with this template with the new safe names, e.g. I think we want the UX for these tasks to be

```
just --dotenv-path $(pwd)/.env --justfile ../../../nested.just simulate base
just --dotenv-path $(pwd)/.env --justfile ../../../nested.just simulate base-council
```

Reply:
IIUC, that would be here: https://github.com/ethereum-optimism/superchain-ops/blob/b2ffccae2c2f162f0196447511bcd053a871e47f/src/improvements/script/get-safe.sh#L18